src/libs/installer/packagemanagercore_p.cpp

```diff
@@ -2040,7 +2052,7 @@ void PackageManagerCorePrivate::registerMaintenanceTool()
     settings.setValue(QLatin1String("DisplayIcon"), maintenanceTool);
     settings.setValue(scPublisher, m_data.value(scPublisher));
     settings.setValue(QLatin1String("UrlInfoAbout"), m_data.value(QLatin1String("Url")));
-    settings.setValue(QLatin1String("Comments"), m_data.value(scTitle));
+    // settings.setValue(QLatin1String("Comments"), m_data.value(scTitle));
```

Comment:
Could you remove rather than comment out?

Reply:
Better?